aim/api/resource.py

```diff
@@ -66,6 +85,8 @@ class BridgeDomain(ResourceBase):
     # Attrbutes completely managed by the DB (eg. timestamps)
     db_attributes = []
 
+    _aci_mo_name = 'fvBD'
```

Comment:
In that case, let's merge this as is. Let's take a decision on whether we want to move the DN calculation in the converter for concern separation though, and revisit this once we do.